engine/storage/datamotion/src/main/java/org/apache/cloudstack/storage/motion/StorageSystemDataMotionStrategy.java

```diff
@@ -2897,4 +2899,27 @@ private CopyCmdAnswer performCopyOfVdi(VolumeInfo volumeInfo, SnapshotInfo snaps
 
         return copyCmdAnswer;
     }
+
+    protected Boolean supportStoragePoolType(StoragePoolType storagePoolTypeToValidate, StoragePoolType... extraAcceptedValues) {
+        List<StoragePoolType> values = new ArrayList<>();
+
+        values.add(StoragePoolType.NetworkFilesystem);
+        values.add(StoragePoolType.SharedMountPoint);
```

Comment:
is this true for all hypervisors/storage backends? I know this is a new method only called from the KVMNonManaged, but that is not reflected in the name or protected by encapsulation. (genuine question)

Reply:
Actually KVM is the only hypervisor that enables `SharedMountPoint` as an option, so it is true to all hypervisors that has it enabled.